app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -0,0 +1,31 @@
+package org.simple.clinic.bp.history
```

Comment:
Reword the commit message to match our conventions?

Ex: `Set up the blood pressure history screen`